### PR TITLE
Hive/master: use stock nightlies

### PR DESCRIPTION
**Pull request by 2uasimojo**
...instead of on-the-fly builds.

I think.

[HIVE-2234](https://issues.redhat.com//browse/HIVE-2234)

### CODE REVIEW

**Comment by 2uasimojo** [comment]
/pj-rehearse pull-ci-openshift-hive-master-e2e pull-ci-openshift-hive-master-e2e-pool

File: ci-operator/jobs/openshift/hive/openshift-hive-master-postsubmits.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ postsubmits:
     labels:
       ci-operator.openshift.io/is-promotion: "true"
       ci.openshift.io/generator: prowgen
+      job-release: "4.13"
```
Comment (suhanime) [inline ci-operator/jobs/openshift/hive/openshift-hive-master-postsubmits.yaml:14]: Confused about what side-effect added these - probably unrelated.

**Review by suhanime — COMMENTED** [review]
/lgtm

**Pull request merged**